Documentation/git-clone.txt

```diff
@@ -15,7 +15,7 @@ SYNOPSIS
 	  [--dissociate] [--separate-git-dir <git dir>]
 	  [--depth <depth>] [--[no-]single-branch] [--no-tags]
 	  [--recurse-submodules[=<pathspec>]] [--[no-]shallow-submodules]
-	  [--[no-]remote-submodules] [--jobs <n>] [--] <repository>
+	  [--[no-]remote-submodules] [--jobs <n>] [--sparse] [--] <repository>
```

Comment:
Since I can't comment on the commit message, I'll park this here.

On the "clone: add --sparse mode" commit:
Where you mention "README" you might also mention files like .gitignore
and friends.

Reply:
And `.gitattributes`!